pandas/plotting/_core.py

```diff
@@ -783,7 +783,7 @@ class PlotAccessor(PandasObject):
     _kind_aliases = {"density": "kde"}
     _all_kinds = _common_kinds + _series_kinds + _dataframe_kinds
 
-    def __init__(self, data):
+    def __init__(self, data) -> None:
```

Comment:
I am pretty sure that not annotating this was a deliberate decision in the past.

cc @simonjayhawkins did this change over the last few weeks?

Reply:
As far as I can see, this line was added on 03/07/2019([pull-request](https://github.com/pandas-dev/pandas/pull/27009)) without any additional comments. And this line also has not changed since those times.

Reply:
> I am pretty sure that not annotating this was a deliberate decision in the past.
> 
> cc @simonjayhawkins did this change over the last few weeks?

As far as I can see, this line was added on 03/07/2019(https://github.com/pandas-dev/pandas/pull/27009) without any additional comments. And this line also has not changed since those times.

Reply:
I am talking about init returning None in general

Reply:
Simon mentioned this in https://github.com/pandas-dev/pandas/pull/44677#discussion_r771575699